public/app/plugins/datasource/prometheus/language_provider.ts

```diff
@@ -188,14 +206,10 @@ export default class PromQlLanguageProvider extends LanguageProvider {
     if (labelKeys) {
       suggestions.push({ label: 'Labels', items: labelKeys.map(wrapLabel) });
     } else {
-      refresher = this.fetchSeriesLabels(selector);
+      result.refresher = this.fetchSeriesLabels(selector);
     }
 
-    return {
-      refresher,
-      suggestions,
-      context: 'context-aggregation',
-    };
+    return result;
```

Comment:
I prefer to not modify objects, even if it allows for a cleaner flow like here. This is minor though.